### DISCUSSION TITLE
Macad and a Studio Mac M2

### DISCUSSION THREAD

**Discussion by tgoy99** [Macad3D/Macad3D#68] [Q&A]
Is it possible to add Macad to a Studio Mac computer?  Recently, Macad was installed into Crossovers, in the bottle, the program opened ok, however, the samples in the program would not open.   Is it possible that some setting was missed  in Macad to get the program to work?
Tim

**Comment by dipts** [comment]
I have tested with the current version of Wine and Crossover on Ubuntu Linux, found the problem at startup and implemented a workaround with 650ac180b92136f06e6819b9139ed60fbae2054f. 

This means that Macad is basically executable, even if I still saw slight artifacts in the GUI. However, as I can only test under a Virtual Machine, I don't know whether it may be due to the graphics emulation of the VM.

I have also sent you a link by email with a version to test. Please check your junk mail folder.

Tobias
